persistence/persistence-api/src/main/java/org/cbioportal/persistence/MutationRepository.java

```diff
@@ -23,6 +23,11 @@ MutationMeta getMetaMutationsInMolecularProfileBySampleListId(String molecularPr
                                                            Integer pageSize, Integer pageNumber,
                                                            String sortBy, String direction);
 
+    List<Mutation> getMutationsInMultipleMolecularProfilesByMutationType(List<String> molecularProfileIds, List<String> sampleIds,
+                                                                         List<Integer> entrezGeneIds, String mutationType, Boolean complementSet, String projection,
```

Comment:
What are the possible values of mutation type and why not use an enum?

Reply:
good catch, @n1zea144! I do not think this method ever be used. Will do some testing tomorrow before removing it. thanks!

Reply:
Hi @n1zea144 , I've done my corrections and committed the changes. Please have your final review. Thank you!